integration_tests/test_snap.py

```diff
@@ -169,3 +170,20 @@ def test_error_on_bad_yaml(self):
             "Issues while validating snapcraft.yaml: found character '\\t' "
             "that cannot start any token on line 13 of snapcraft.yaml",
             str(error.output))
+
+    def test_aliases(self):
```

Comment:
Can you instead put this test in `test_alias.py`? You can probably merge it with the test that's already there.